hydromt_fiat/api/hydromt_fiat_vm.py

```diff
@@ -105,10 +105,10 @@ def build_config_yaml(self):
         if self.exposure_vm.exposure_roads_model:
             config_yaml.setup_exposure_roads = self.exposure_vm.exposure_roads_model
 
-        if self.vulnerability_vm.vulnerability_roads_model:
-            config_yaml.setup_road_vulnerability = (
-                self.vulnerability_vm.vulnerability_roads_model
-            )
+        #if self.vulnerability_vm.vulnerability_roads_model:
+        #    config_yaml.setup_road_vulnerability = (
+        #        self.vulnerability_vm.vulnerability_roads_model
+        #    )
```

Comment:
see my previous comment. Here the if statement should not pass if you don't have self.vulnerability_vm.vulnerability_roads_model right? So you could keep it the code.